modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -1367,6 +1367,7 @@ anonymous-posting=Anonymous Posting
 anonymous-user-configuration-name=Anonymous User
 anonymous-user-segments-cache-expiration-time-description=Define the time, in seconds, in which the anonymous segment cache is cleared.
 anonymous-user-segments-cache-expiration-time-name=Anonymous User Segments Cache Expiration Time
+another-node-is-already-using-this-id=Another node is already using this ID. Please, define a different ID.
```

Comment:
Hey @rbohl, Could you please take a look at this error message and give your feedback on its clarity? Thx Russ

/cc @naraandrade

Reply:
I think this is slightly clearer and matches more closely with similar keys I see in master:

> A node with that ID already exists. Please enter a unique ID.